### DISCUSSION TITLE
How should I made small ProgressBar?

### DISCUSSION THREAD

**Discussion by Orholt** [AvaloniaUI/Avalonia#6330] [Q&A] [answered]
Hi, I tried to make ProgressBar component with code looking like this:
```
	<Panel>
	  <ProgressBar Height="200" Value="5" Orientation="Vertical" Width="20" Minimum="0"
	  Maximum="10"></ProgressBar>
	</Panel>
```

And I realized I can't make it smaller than **200**, any ideas why? Or how should I achieve it?
Or maybe this is some kind of issue?

![ProgressBars](https://user-images.githubusercontent.com/42500203/127655376-de00eedf-e16a-4e14-9c26-266f1e9923f3.png)

**Comment by maxkatz6** [comment, marked as answer]
Check min width/height

**Reply by Orholt** [reply]
Thanks it worked when I set the MinHeight to 100 or lower
